java/src/server/commands/moves/RollNumberCommand.java

```diff
@@ -9,21 +13,32 @@
  * @author Joel Bradley
  */
 public class RollNumberCommand implements ICommand {
+    private int gameId;
+    private int value;
+    private int playerIndex;
+    private IFacade facade;
 
     /**
      * Constructor
      */
-    public RollNumberCommand() {
-
+    public RollNumberCommand(IFacade facade, RollNumberDTO roll) {
+        this.facade = facade;
+        this.gameId = roll.getGameId();
+        this.value = roll.getValue();
+        this.playerIndex = roll.getPlayerIndex();
     }
 
     /**
      * Communicates with the ServerFacade to carry out the Roll Number command
-     * @return IDTO
+     * @return JsonObject
      */
     @Override
-    public IDTO execute() {
-        return null;
+    public IDTO execute() throws CommandExecutionFailedException {
+        try {
+            return facade.rollNumber(gameId, playerIndex, value);
```

Comment:
I'm confused, that is in essence what I'm doing...I think. The command gets passed in the gameId and the DTO. I just extract what I need from the DTO in the constructor instead of having to call "dto.getPlayerIndex()" within my rollNumber function call. It looks cleaner IMO.